tests/unit/fs/test_tree.py

```diff
@@ -39,11 +39,9 @@ def test_get_cloud_fs_validate(tmp_dir, dvc):
         default=False,
     )
 
-    assert get_cloud_fs(dvc, name="base")[1] == {"host": "example.com"}
-    assert get_cloud_fs(dvc, name="first")[1] == {
```

Comment:
Excuse me, any difference between these two forms?

Reply:
the difference is that, now we check the precise key instead of the whole dict which might change in time depending to the config.